### PR TITLE
print a nicer error on timeout/cancel

### PR DESCRIPTION
**Pull request by Stebalien**
*Don't* say "context canceled". Just say "canceled".

fixes https://github.com/ipfs/go-ipfs/issues/3785

### CODE REVIEW

File: cli/responseemitter.go [hunk 1]
```diff
@@ -63,19 +63,16 @@ func (re *responseEmitter) SetLength(l uint64) {
...
-		e = &cmdkit.Error{
-			Message: err.Error(),
-		}
+	case context.Canceled:
+		msg = "canceled"
+	case context.DeadlineExceeded:
+		msg = "timed out"
+	default:
+		msg = err.Error()
 	}
```
Comment (Stebalien) [inline cli/responseemitter.go]: We don't need a `cmdkit.Error`, we just need the error message.

**Comment by Stebalien** [comment]
This now prints:

```
Error: canceled
```

When a user hits `^C` and:

```
Error: timed out
```

When the command times out.

**Comment by djdv** [comment]
I like this change but wonder if prepending "request" is necessary.
>Error:  request canceled
Error: request timed-out

vs
>Error:  canceled
Error: timed-out

**Comment by Stebalien** [comment]
We can bikeshed this in a followup PR.

**Pull request merged**